webpack/components/Table/EmptyStateMessage.js

```diff
@@ -55,11 +77,47 @@ const EmptyStateMessage = ({
         <EmptyStateBody>
           {emptyStateBody}
         </EmptyStateBody>
+        {showPrimaryAction &&
+          <Button>
+            <a href={primaryActionLink} style={{ color: 'white', textDecoration: 'none' }}>{primaryActionTitle}</a>
+          </Button>}
+        {showSecondaryAction &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link">
+              <a href={secondaryActionLink} style={{ textDecoration: 'none' }}>{secondaryActionTitle}</a>
+            </Button>
+          </EmptyStateSecondaryActions>
+        }
+
+        {searchIsActive && !filtersAreActive &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link" onClick={clearSearch}>
+              Clear search
```

Comment:
These need to be translated
```suggestion
              {__('Clear search')}
```